### ISSUE TITLE
Add option for Met Office - IMO - BGS MoU colour scheme

### ISSUE THREAD

**Issue by volcan01010** [BritishGeologicalSurvey/ash-model-plotting#2]
### Summary

As a member of staff at BGS, I want to be able to make plots using the colour schemes agreed in the MoU.

### Description

The branch `new_color_map` includes the MoU colour scheme.  For it to work, it depends on the cubes being stored with the correct units.  These should be recorded on this ticket.

A possible implementation of the colour map switching would be to replace the `vaac_colours` flag with a colour map option.  It could choose from `vaac`, `iceland_uk_mou` or other builtin options.

### Acceptance criteria

+ [ ] Unit tests show that total_deposition is presented in standard units (g/m2)
+ [ ] Unit tests show that total_column is presented in standard units
+ [ ] Unit tests show that air_concentration is presented in standard units
+ [ ] An option is added to the plotting functions allowing the MoU colours to be used

**Comment by FabioDioguardi** [comment]
To my knowledge, units should be:

- Total deposition: g/m2

- Total column: g/m2

- Air concentration: g/m3

**Comment by FabioDioguardi** [comment]
I agree with the implementation of the "colour_map" option in the arguments. 
We could then allow the following options:
- "vaac_colours": the question is: what about deposition plots? I do not think the VAAC colours are designed for the deposition plots
- "iceland_uk_mou": the question is: what about the air concentration maps? We do not have a MoU standard for these, for the moment I have used the same colours and fixed different levels. For the levels I suggest a solution described in the next point
- "automatic": we decide a standard colourmap among the built-in ones; about the levels, I think we should fix the minimum and maximum to have consistent and comparable plots; i.e., the same levels for all the Air concentration plots, the same for all the deposition plots and so on. These can be defined at each simulation by searching for the maximum among all the results (the minimum can be set to 0 always).  
We should also decide the number of levels (10? 15?).

**Comment by FabioDioguardi** [comment]
HYSPLIT Total_column plots are now displaying "unknown" in the units label (I am referring to the operational simulations)
